scripts/pi-hole/js/footer.js

```diff
@@ -97,7 +103,7 @@ function countDown() {
     ena.text("Enable Blocking (" + secondsTimeSpanToHMS(seconds) + ")");
   } else {
     ena.text("Enable Blocking");
-    piholeChanged("enabled");
+    piholeChanged("enabled", null);
```

Comment:
Nit-picky comment: As `timer` has a default declaration of `null` in the parameters for `piholeChanged`, does `null` need to be explicitly added to the function call here?